packages/calcite-components/src/components/popover/popover.tsx

```diff
@@ -145,6 +146,9 @@ export class Popover
   /** Specifies the heading level of the component's `heading` for proper document structure, without affecting visual styling. */
   @property({ type: Number, reflect: true }) headingLevel: HeadingLevel;
 
+  /** Specifies whether the first tabbable child element will be focused when the popover is opened. */
+  @property({ reflect: true }) initialFocus: FocusTargetOrFalse;
```

Comment:
✨ 💡 Cool 💡 ✨ find! I didn't know about this option.

Reply:
Do we need this to be a public property? I'm not sure we want to add a property like this. 

This property also wouldn't be honored if the `setFocus()` method was used so I'm not sure it needs more doc to say its only for focus trapping.

Reply:
If there's a way to provide an "internal" hook to this option, that would also work for this use case; it doesn't have to be exposed publicly.  Also, `input-date-picker` currently utilizes this option in the `connectFocusTrap` function that it consumes directly, but probably makes sense to refactor it at some point to consume `calcite-popover` internally.

Reply:
I would suggest marking this property as `@internal/@private` in order to publicly document it if we do go this route.

I'm not sure this is the best route because it deviates from how all the other focus trapping works. In general, when you open something you focus trap it and place focus inside of what is being focus trapped.

However, in this case, you are creating the focus trap but the user isn't within the trapped component yet so its really doing nothing. 

cc @jcfranco

Reply:
Ideally, you wouldn't need to ever set focus in the popover. focus would always remain on the input and use keyboard arrows/enter/space for handling things. This is what the `action-menu` component does.

Reply:
Just replaced the doc wording with `@internal`.